### DISCUSSION TITLE
The verification token is invalid or has already been used. Please retry the flow.

### DISCUSSION THREAD

**Discussion by hugotiburtino** [ory/kratos#2796] [Show and tell]
Just wanted to note that kratos at version 0.10.1 is not creating the verification token after registration (maybe there was an issue about that, but I can't find any). You can be sure of that if you create a new account (of course, with an identity schema in which email is present and verification via email, too) and take a look at the table `identity_verification_tokens`, where you won't find the corresponding token (just for completeness: I'm sure the email client isn't opening the link, since I'm testing it with mailslurper). If user clicks on verification link, they get 'The verification token is invalid or has already been used. Please retry the flow.'.

Anyways, in the next version it'll be solved.

**Comment by vinckr** [comment]
Hello @hugotiburtino 
Sorry for the late answer, and thanks for the report!
Is this still an issue for you or did it get solved with the next release?

**Reply by hugotiburtino** [reply]
it was solved
